sympy/core/tests/test_complex.py

```diff
@@ -210,3 +210,8 @@ def test_issue_5429():
 def test_issue_4124():
     from sympy import oo
     assert expand_complex(I*oo) == oo*I
+
+def test_issue_10897():
```

Comment:
you can remove this test since the same code is already in ``test_complexes.py``